discord/ui/text_display.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, TypeVar
+
+from ..components import TextDisplay as TextDisplayComponent
+from ..components import _component_factory
+from ..enums import ComponentType
+from .item import Item
+
+__all__ = ("TextDisplay",)
+
+if TYPE_CHECKING:
+    from ..types.components import TextDisplayComponent as TextDisplayComponentPayload
+    from .view import View
+
+
+T = TypeVar("T", bound="TextDisplay")
+V = TypeVar("V", bound="View", covariant=True)
+
+
+class TextDisplay(Item[V]):
+    """Represents a UI TextDisplay.
+
+    .. versionadded:: 2.7
+
+    Parameters
+    ----------
+    content: :class:`str`
+        The text display's content.

```

Comment:
character limit?

Reply:
There is none as far as I am aware, it is 4000 characters but that is the total character limit as well so not sure if that would be a limit on this field

Reply:
we have historically not cared about character limits, since they can change